pkgs/top-level/python-packages.nix

```diff
@@ -316,6 +316,8 @@ in {
     callPackage = pkgs.callPackage;
   };
 
+  pyprof2calltree = callPackage ../development/python-modules/pyprof2calltree { };
```

Comment:
That's the wrong location. Applications belong into `all-packages.nix`. See https://nixos.org/nixpkgs/manual/#building-packages-and-applications. Also, your derivation should probably be put into `pkgs/tools/...`

Reply:
Well, it's a development tool, so maybe I should rather move the derivation to pkgs/development/tools/pyprof2calltree ?

Reply:
:+1: